husky_gazebo/CMakeLists.txt

```diff
@@ -4,13 +4,13 @@ project(husky_gazebo)
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED husky_gazebo_plugins)
```

Comment:
You don't need this; it's a run not build dependency.

In any case, it would be `find_package(catkin REQUIRED COMPONENTS husky_gazebo_plugins)`